### PR TITLE
Add tests to check markdown component in return from backend

### PR DESCRIPTION
**Pull request by rgarg1**

### CODE REVIEW

File: EE_API_automation/pytest/src/test_planner.py [hunk 1]
```diff
@@ -672,7 +689,8 @@ def test_edit_workitem_desc(self):
             r = req.patch(url, headers=request_detail.headers_default, json=f)
             ##Analyze the response
             assert r.status_code == 200
-            desc_rendered = str(helpers.extract_value("data.attributes.\"system.description.rendered\"", r))
+            assert helpers.extract_value("data.attributes.\"system.description.markup\"", r) == "Markdown"
+            desc_rendered = str(helpers.extract_value("data.attributes.\"system.description.rendered\"", r)).strip()
```
Comment (jarifibrahim) [inline EE_API_automation/pytest/src/test_planner.py]: There should be an assert after this line. Where are we checking the `desc_rendered` value?

File: EE_API_automation/pytest/src/test_planner.py [hunk 2]
```diff
@@ -317,7 +325,8 @@ def test_edit_workitem_desc(self):
...
```
Comment (jarifibrahim) [inline EE_API_automation/pytest/src/test_planner.py]: Missing assert statement

File: EE_API_automation/pytest/src/test_planner.py (hunk 1 above)
Comment (rgarg1) [reply EE_API_automation/pytest/src/test_planner.py]: line 694 asserts the check @jarifibrahim 

assert wi_new_desc in desc_rendered

File: EE_API_automation/pytest/src/test_planner.py (hunk 2 above)
Comment (rgarg1) [reply EE_API_automation/pytest/src/test_planner.py]: line 330 asserts @jarifibrahim

**Pull request merged**